test/fuzzer/pedro/having_subquery_failed_to_bind.test

```diff
@@ -9,6 +9,10 @@ statement error
 SELECT sum(c2) c0 FROM (SELECT 1) t1(c2) HAVING (SELECT c0);
 ---
 cannot reference alias
+----
+Parser Error: syntax error at or near "cannot"
+LINE 3: cannot reference alias
+        ^
```

Comment:
Does this make a sense? It's the intended error not arleady "cannot reference alias"?

This is repeated in few other places, likely all places where you add a "LINE X:" are to be removed.

Reply:
Yes you're entirely correct
The reason this was caught is because it doesn't use the correct syntax to provide a statement error

Note `---` instead of `----`